app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -224,6 +224,12 @@ const ClaimScreen: React.FC = () => {
       <ScrollView style={commonStyles.screenContent}>
         <View>
           <StatusBar hidden={false} />
+          {claim.type === "MobileCredential" ? (
```

Comment:
&& so dont have to put null?

Reply:
we got advice a while ago to not use && inline in react native and instead use the ? : null syntax to avoid some fringe errors

Reply:
Cool.